webserver/app/api.py

```
@@ -0,0 +1,31 @@
+from fastapi import FastAPI, HTTPException
+from .tasks import process_audio
+from .schemas import AddTaskRequest
+from .config import settings
+from .celery_app import celery, check_queue_empty, shutdown_gcp_instance
+
+app = FastAPI()
+
+@app.post("/add_task")
+async def add_task(request: AddTaskRequest):
```

Ruff (0.11.9)

Missing return type annotation for public function `add_task`

(ANN201)